mortar/src/main/java/mortar/MortarScope.java

```diff
@@ -30,8 +30,20 @@
   public static final String DIVIDER = ">>>";
   public static final String SERVICE_NAME = MortarScope.class.getName();
 
+  /**
+   * Retrieves a MortarScope from the given context. If none is found, retrieves a MortarScope from
+   * the application context.
+   *
+   * @return null if no scope is found in either the given context or the application context
+   */
   public static MortarScope getScope(Context context) {
-    return (MortarScope) context.getSystemService(MortarScope.SERVICE_NAME);
+    //noinspection ResourceType
+    Object scope = context.getSystemService(SERVICE_NAME);
+    if (scope == null) {
+      //noinspection ResourceType
+      scope = context.getApplicationContext().getSystemService(SERVICE_NAME);
```

Comment:
How does this work? If ContextWrapper.mBase is null, then context.getApplicationContext() will NPE. What fields are set and not set in the activity precisely? Just to be clear: `context.getApplicationContext()` goes through the activity, always.